src/Castle.Services.Logging.NLogIntegration/Castle.Services.Logging.NLogIntegration.csproj

```diff
@@ -25,7 +25,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="NLog" Version="4.5.0" />
+		<PackageReference Include="NLog" Version="4.7.15" />
```

Comment:
This is an unrelated change; please revert it.

Reply:
Think I was having culture-issue with the old build pipeline (NLog v4.5 doesn't support `InvariantGlobalization`). Have now reverted it again.